configuration/micro_kernel_trait.rst

```diff
@@ -350,4 +350,4 @@ Then see webpage in browser:
 
 Hey, that looks a lot like a *traditional* Symfony application! You're right: the
 ``MicroKernelTrait`` *is* still Symfony: but you can control your structure and
-features quite easily.
+features in a lightweight way.
```

Comment:
lighter way? - 'lightweight way' sounds a bit strange when you speak it out lout 😄